roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -395,3 +395,30 @@ schema_registry_additional_system_admins: "{{rbac_component_additional_system_ad
 ksql_additional_system_admins: "{{rbac_component_additional_system_admins}}"
 kafka_connect_additional_system_admins: "{{rbac_component_additional_system_admins}}"
 control_center_additional_system_admins: "{{rbac_component_additional_system_admins}}"
+
```

Comment:
Can you document each of these variables? 

How to do this was never explained lol, so I will here. Essentially you need to put 
```
### Variable description
variable: val
```

And then run `python docs.py` which will write everything in markdown